client/flutter/lib/api/content_loading.dart

```diff
@@ -35,6 +36,18 @@ class ContentLoading {
         return await _loadFromNetwork(name, languageAndCountry);
       } catch (err) {
         print("Network bundle for $languageAndCountry not found: $err");
+        if (err is ContentBundleVersionException) {
```

Comment:
`on ContentBundleVersionException catch (err)` would be better

Reply:
It does need to handle the exception and print the message for both... does that fall through?  (Not familiar enough with Dart exception handling :) )

Reply:
Not sure what you're after so here's a DartPad example to play with https://dartpad.dev/a413da20d821e80d10f78540cec452fe

Reply:
Pat is right let's keep it this way